docs/setup/datalogger.md

```diff
@@ -24,6 +24,51 @@ After:
 
 After this your ShineLink/Wifi Dongle will start pushing data to Grott (and optionally more frequently than default also).
 
+## ShineWiFi-X/S Re-configuration
+
+To reconfigure your ShineWiFi-X/S datalogger, you need to use the ShinePhone app and have your datalogger already added to your plant. (This guide and photos are based on Android.)
+
+1. **Start the ShinePhone App:**  
```

Comment:
Please can you change all of the bullet point numbers to `1.` Github will auto-re-number them when the page is published, it means that if there's any modifications in the future nobody has to go through and change all the numbers.

Reply:
They are now updated as requested. 👍